saleor/graphql/product/mutations/products.py

```diff
@@ -525,10 +527,38 @@ def clean_attributes(
         )
         return attributes
 
+    @staticmethod
+    def parse_description_json_to_string(description):
+        string = ""
+        blocks = description.get("blocks")
+        if not blocks or not isinstance(blocks, list):
+            return ""
+
+        for block in blocks:
+            block_type = block["type"]
+            if block_type == "list":
+                for item in block["data"].get("items"):
+                    if not item:
+                        continue
+                    string += clean_text_data(item)
+            else:
+
+                text = block["data"].get("text")
+                if not text:
+                    continue
+                string += clean_text_data(text)
+        return string
+
     @classmethod
     def clean_input(cls, info, instance, data):
         cleaned_input = super().clean_input(info, instance, data)
 
+        description = cleaned_input.get("description")
+        if description:
```

Comment:
What if the description is empty? One may want to clear the content of the description field and it should result in clearing the `description_search` field. Would that happen with the current implementation?

Reply:
I updated this functionality to clear `description_search` when there is no `description` in payload